common/src/main/scala/net/psforever/packet/game/ChatMsg.scala

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2016 PSForever.net to present
+package net.psforever.packet.game
+
+import net.psforever.packet.{GamePacketOpcode, Marshallable, PacketHelpers, PlanetSideGamePacket}
+import scodec.Codec
+import scodec.codecs._
+
+// ChatMessageTypes were reversed by:
```

Comment:
Can you change these // comments to
```
/** Some brief comment
  *
  * Some longer explanation.
  */
```
That way they get placed in generated documentation